model.py

```diff
@@ -0,0 +1,64 @@
+from sqlalchemy import Column, ForeignKey, Integer, String, DateTime
+from sqlalchemy.orm import relationship
+from sqlalchemy.ext.declarative import declarative_base
+
+# Create the base class for declarative models
+Base = declarative_base()
```

Comment:
It's better to define Base be a defined as a class and inherit from DeclarativeBase